.ci/jenkins/lib/build-wheel-matrix.yaml

```
@@ -0,0 +1,227 @@
+  namespace: nbu-swx-nixl
+  # 16GB memory and 16 CPU cores for parallel compilation and large builds
+  limits: '{memory: 16Gi, cpu: 16000m}'
+  requests: '{memory: 16Gi, cpu: 16000m}'
+
+# =============================================================================
+# Docker Build Configuration
+# =============================================================================
+# Multi-stage Docker build using only the 'wheel-base' stage for CI
+# This provides a clean build environment without pre-built wheels
+runs_on_dockers:
+  - { name: "podman", url: "quay.io/podman/stable:v5.7.1", privileged: true }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)